### PR TITLE
fixes Spring section

### PR DESCRIPTION
**Pull request by otaviojava**

### CODE REVIEW

File: docs/src/languages/java/frameworks.md [hunk 1]
```diff
@@ -75,18 +75,24 @@ The [Spring Framework](https://spring.io/projects/spring-framework) provides a c
...
+| [Spring MVC and Spring Data MongoDB](https://platform.sh/blog/2019/spring-mvc-and-mongodb-a-match-made-in-platform.sh-heaven/) | [Source](https://github.com/platformsh/java-quick-start/tree/master/spring/spring-mvc-maven-mongodb) |
+| [Spring Boot and Spring Data JPA](https://platform.sh/blog/2019/java-hello-world-at-platform.sh/) | [Source](https://github.com/platformsh-templates/spring-boot-maven-mysql) |
 
 ## Tomcat
 
 Apache Tomcat is an open-source implementation of the Java Servlet,  JavaServer Pages, Java Expression Language and WebSocket technologies.
 
+
+
+> TIP: By default, Spring Boot provides an embedded Apache Tomcat build. Therefore, if you want to use Tomcat with Spring, check the Spring section.
```
Comment (chadwcarlson) [inline docs/src/languages/java/frameworks.md:90]:
```suggestion
{{< note title="Tip" >}}
By default, Spring Boot provides an embedded Apache Tomcat build. Therefore, if you want to use Tomcat with Spring, check the Spring section.
{{< /note >}}
```

File: docs/src/languages/java/frameworks.md [hunk 2]
```diff
@@ -75,18 +75,20 @@ The [Spring Framework](https://spring.io/projects/spring-framework) provides a c
...
+| [Spring Data ElasticSearch and Spring Data Sorl](https://platform.sh/blog/2019/elasticsearch-vs-solr-have-both-with-spring-data-and-platform.sh/) | [Elasticsearch](https://github.com/platformsh/java-quick-start/tree/master/spring/spring-mvc-maven-elasticsearch) and [Solr](https://github.com/platformsh/java-quick-start/tree/master/spring/spring-mvc-maven-solr) |
+| [Spring MVC and Spring Data MongoDB](https://platform.sh/blog/2019/spring-mvc-and-mongodb-a-match-made-in-platform.sh-heaven/) | [Source](https://github.com/platformsh/java-quick-start/tree/master/spring/spring-mvc-maven-mongodb) |
+| [Spring Boot and Spring Data JPA](https://platform.sh/blog/2019/java-hello-world-at-platform.sh/) | [Source](https://github.com/platformsh-templates/spring-boot-maven-mysql) |
 
 ## Tomcat
 
 Apache Tomcat is an open-source implementation of the Java Servlet,  JavaServer Pages, Java Expression Language and WebSocket technologies.
 
+> TIP: By default, Spring Boot provides an embedded Apache Tomcat build. Therefore, if you want to use Tomcat with Spring, check the Spring section.
```
Comment (Crell) [inline docs/src/languages/java/frameworks.md:90]: We don't use this style for tips anywhere else, I think.

Comment (Crell) [reply docs/src/languages/java/frameworks.md:90]: Chad, what's the new Hugo syntax for a "note" box?

Comment (otaviojava) [reply docs/src/languages/java/frameworks.md]: He already post it with ```{{< note >}}```
Thank you @chadwcarlson

Comment (otaviojava) [reply docs/src/languages/java/frameworks.md]: btw, is there one way to put a link that goes to the same page?
E.g.: `Spring section` moves to the Spring title section

Comment (Crell) [reply docs/src/languages/java/frameworks.md]: Just link to `#the-anchor`.  If it's within the same page that should be all that's needed.

Comment (otaviojava) [reply docs/src/languages/java/frameworks.md]: Thank you

**Pull request merged**